packages/lib/src/vis/shared/VisGroup.tsx

```diff
@@ -0,0 +1,31 @@
+import type { ReactNode } from 'react';
+
+import { useAxisSystemContext } from './AxisSystemProvider';
+
+interface Props {
+  children: ReactNode;
+}
+
+function VisGroup(props: Props) {
+  const { children } = props;
+  const { abscissaConfig, ordinateConfig, visSize } = useAxisSystemContext();
+  const { width, height } = visSize;
+
+  const sx =
+    ((abscissaConfig.flip ? -1 : 1) * width) /
+    (abscissaConfig.visDomain[1] - abscissaConfig.visDomain[0]);
+  const sy =
+    ((ordinateConfig.flip ? -1 : 1) * height) /
+    (ordinateConfig.visDomain[1] - ordinateConfig.visDomain[0]);
+  const x = 0.5 * (abscissaConfig.visDomain[0] + abscissaConfig.visDomain[1]);
+  const y = 0.5 * (ordinateConfig.visDomain[0] + ordinateConfig.visDomain[1]);
```

Comment:
I would either include the full computation of the position coordinates in `x` and `y`, or rename these `centerX` and `centerY`.